assets/src/amp-validation/amp-validation-error-edit-screen.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+import { __ } from '@wordpress/i18n';
+
+domReady( () => {
+	const errorDetailList = document.querySelector( 'dl.detailed' );
+	addAdditionalErrorDetails( errorDetailList );
```

Comment:
As opposed to using JS for this, couldn't the new `dt`/dd` be added in PHP by injecting them before the first `<dt>`?

Reply:
Done in https://github.com/ampproject/amp-wp/pull/4382/commits/f571c17a827cc209c63bbf55c562667568b209ea.